client/sections.js

```diff
@@ -122,20 +122,18 @@ if ( config.isEnabled( 'reader' ) ) {
 	} );
 }
 
-if ( config.isEnabled( 'post-editor' ) ) {
-	editorPaths = [ '/post' ];
+editorPaths = [ '/post' ];
 
-	if ( config.isEnabled( 'post-editor/pages' ) ) {
-		editorPaths.push( '/page' );
-	}
-
-	sections.push( {
-		name: 'post-editor',
-		paths: editorPaths,
-		module: 'post-editor'
-	} );
+if ( config.isEnabled( 'post-editor/pages' ) ) {
```

Comment:
I was going to clean up that flag in a subsequent PR, but will remove this instance of it here.